base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x, T)
+    mod(x, T)
+    %(x, T)
+
+For an integer type `T`, finds `z::T` such that `x` ≡ `z` (mod n), where n is the
+cardinality of `T` and `z` is restricted to the interval of values representable by `T`.
```

Comment:
Would it be clearer to explain this by saying that `x % T` returns `y :: T` such that `x` ≡ `y` in the native modular arithmetic of the type `T`? For example, `x % Int8` returns `y :: Int8` such that `x` ≡ `y` (mod 256) – i.e. one of the representatives from `-128` through `127`.